drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -378,6 +378,13 @@ static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len)
             res = sizeof(int8_t);
             break;
 
+        case NETOPT_AUTOACK :
+            assert(max_len >= sizeof(netopt_enable_t));
+            uint8_t tmp = at86rf2xx_reg_read(dev, AT86RF2XX_REG__CSMA_SEED_1);
+            *((netopt_enable_t *)val) = tmp & AT86RF2XX_CSMA_SEED_1__AACK_DIS_ACK ? false : true;
+            res = sizeof(netopt_enable_t);
+            break;
```

Comment:
Mhhh... Though it might be cleaner to move this to `at86rf2xx_get_option()` to be symmetric to the setter.

Reply:
Sorry, just realized that there is no such function...